Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -166,6 +166,19 @@ public ICollection<BoundPeer> Neighbors(Address target, int k)
             return peers;
         }
 
+        public ICollection<BoundPeer> SpecificPeer(Address specificPeer)
+        {
+            var sorted = _buckets
+                .Where(b => !b.IsEmpty())
+                .SelectMany(b => b.Peers)
+                .ToList();
+
+            var peers = new List<BoundPeer>();
+            peers.Add(sorted.Where(peer => peer.Address.Equals(specificPeer)).FirstOrDefault());
+
+            return peers;
```

Comment:
I recommand you to use `RoutingTable.GetBucketIndexOf()` method to find specific peer.

Reply:
Just so I completely understanding what you're looking for.  We are provided a peer to look for and we want to find where it resides?  Instead of what I implemented of, we are provided an address and returning a peer with that address.

Reply:
Your understanding is correct—we have to find the peer with the specific address. But it's implementation is not accurate so it may fail to find peer with the address even it is on the network. 
Since Kademlia is one of protocol to implement [DHT (Distribute Hash Table)](https://en.wikipedia.org/wiki/Distributed_hash_table), a peer may not contain a specific peer even it is on network. So we have to recursively find the specific peer by asking closest peer to target address which peer-to-look has in each table. Since Kademlia saves closes peer more specifically, it can finally reach the target address if it is on the network.
Feel free to ask us if you have more questions.

Reply:
Okay, I've tried a different approach.  Could you please review the PR and let me know your thoughts on this?